### DISCUSSION TITLE
Alphabet side search bar

### DISCUSSION THREAD

**Discussion by ile123** [zyrouge/symphony#423] [Ideas]
Well the title says it all really, it would be cool if this could be implemented into the app. It will make searching for a specific song much easier when you can immediately jump to all songs that start with that letter.

**Comment by zyrouge** [comment]
Do you mean scrollbar with alphabets? Currently, it is implementable in Compose without manually messing around with it's scroll mechanism.

**Reply by zyrouge** [reply]
Being track in #426.
